include/pros/apix.h

```diff
@@ -30,6 +30,122 @@
 #pragma GCC diagnostic pop
 #include "pros/serial.h"
 
+
+/******************************************************************************/
+/**                            RTOS CONFIGURATION                            **/
+/******************************************************************************/
+
+struct xSTATIC_LIST_ITEM {
+	uint32_t xDummy1;
+	void *pvDummy2[ 4 ];
+};
+typedef struct xSTATIC_LIST_ITEM StaticListItem_t;
+
+struct xSTATIC_MINI_LIST_ITEM {
+	uint32_t xDummy1;
+	void *pvDummy2[ 2 ];
+};
+typedef struct xSTATIC_MINI_LIST_ITEM StaticMiniListItem_t;
+
+typedef struct xSTATIC_LIST {
+	uint32_t uxDummy1;
+	void *pvDummy2;
+	StaticMiniListItem_t xDummy3;
+} StaticList_t;
+
+#define portUSING_MPU_WRAPPERS                  0
+#define configMAX_TASK_NAME_LEN                 ( 32 )
+#define portSTACK_GROWTH                        ( -1 )
+#define configRECORD_STACK_HIGH_ADDRESS         0
+#define portCRITICAL_NESTING_IN_TCB             0
+#define configUSE_TRACE_FACILITY                1
+#define configUSE_MUTEXES                       1
+#define configUSE_APPLICATION_TASK_TAG          0
+#define configNUM_THREAD_LOCAL_STORAGE_POINTERS 2
+#define configGENERATE_RUN_TIME_STATS           1
+#define configUSE_NEWLIB_REENTRANT              1
+#define configUSE_TASK_NOTIFICATIONS            1
+#define configSUPPORT_STATIC_ALLOCATION         1
+#define configSUPPORT_DYNAMIC_ALLOCATION        1
+#define INCLUDE_xTaskAbortDelay                 1
+#define configUSE_QUEUE_SETS                    0
```

Comment:
I don't think these should be exposed to the end user. I think manually evaluating the macros would be best. Also if we change these in the future, it would be a binary breaking change. I don't think it's a big issue, but it's worth mentioning.

Reply:
In order to create static_task_s_t, I evaluated the final value for the macros here. Is that what `manually evaluating` mean or do you what to remove all config macro to the end-user?

Reply:
Okay, I removed all the config macros.